src/postinstall/prompts.js

```diff
@@ -12,36 +12,36 @@ var prompts = [
     default: 'App.js',
     type: 'string',
     required: false,
-    name: 'entry'
+    name: 'entry',
+  },
+  {
+    description: chalk.cyan('Where does (or will) your index.html file live?'),
+    default: 'index.html',
+    type: 'string',
+    required: false,
+    name: 'index',
   },
   {
     description: chalk.green('Is there a specific name you want me to send your code after I compile it, if so what is it?'),
     default: 'dist',
     type: 'string',
     required: false,
-    name: 'output'
+    name: 'output',
   },
   {
     description: chalk.blue('What port would you like this all to run on?'),
     default: '8080',
     type: 'number',
     required: false,
-    name: 'port'
+    name: 'port',
   },
   {
-    description: chalk.cyan('Where does (or will) your index.html file live?'),
-    default: 'index.html',
+    description: chalk.red('Would you like me to install React for you?'),
+    default: 'no',
     type: 'string',
```

Comment:
Make it easier for yourself. Using a string will just make for horrible case testing.